src/Symfony/Component/Notifier/Bridge/LinkedIn/LinkedInTransport.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LinkedIn;
+
+use Symfony\Component\Notifier\Bridge\LinkedIn\Share\AuthorShare;
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Component\Notifier\Exception\TransportException;
+
+/**
+ * @author Smaïne Milianni <contact@smaine.me>
+ *
+ * @experimental in 5.2
+ *
+ * @see https://docs.microsoft.com/en-us/linkedin/marketing/integrations/community-management/shares/ugc-post-api#sharecontent
+ */
+final class LinkedInTransport extends AbstractTransport
+{
+    protected const PROTOCOL_VERSION = '2.0.0';
+    protected const PROTOCOL_HEADER = 'X-Restli-Protocol-Version';
+    protected const ENDPOINT = 'https://api.linkedin.com/v2/ugcPosts';
```

Comment:
`HOST` instead of `ENDPOINT`

Reply:
@YaFou `HOST` - it is domain, but this constant has value of full url for request, I think what `ENDPOINT` is right name

Reply:
However `getEndpoint` use the HOST constant. So you have to use the `HOST` constant. Look at the Firebase bridge for an example.

Reply:
I think need a review from @fabpot :)

Reply:
updated with `HOST`